lib/email-addresses.js

```diff
@@ -528,7 +528,21 @@ function parse5322(opts) {
             obsGroupList
         )());
     }
-
+    
+    function fromSpec() {
+        return wrap('from', or(
+            mailboxList,
+            addressList
+        )());
+    }
+    
+    function senderSpec() {
```

Comment:
// RFC 6854 2.1.  Replacement of RFC 5322, Section 3.6.2. Originator Fields
// sender = "Sender:" (mailbox / address) CRLF